assets/src/stories-editor/helpers/displayPasteError.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+import { getBlockType } from '@wordpress/blocks';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Helper function to display a paste error message
+ *
+ * @param {string} name Name of block.
+ * @param {Function} createErrorNotice Bring in dispatchable createErrorNotice function.
+ */
+const displayPasteError = ( name, createErrorNotice ) => {
```

Comment:
Instead of having to pass `createErrorNotice`, couldn't this just be turned into a custom React hook?

```js
const useDisplayPasteError = ( name ) => {
	const { createErrorNotice } = useDispatch( 'core/notices' );
	// ...
}
``` 

In `assets/src/stories-editor/components/context-menu/index.js` you could then do:

```js
const displayPasteError = useDisplayPasteError( pastedBlock.name );
displayPasteError();
```